packages/contract/script/pay/DeployDaimoPayAcrossBridger.s.sol

```diff
@@ -0,0 +1,295 @@
+            chainIds[1] = BASE_MAINNET;
+            chainIds[2] = ETH_MAINNET;
+            chainIds[3] = OP_MAINNET;
+            chainIds[4] = POLYGON_MAINNET;
+
+            toTokens[0] = ARBITRUM_MAINNET_USDC;
+            toTokens[1] = BASE_MAINNET_USDC;
+            toTokens[2] = ETH_MAINNET_USDC;
+            toTokens[3] = OP_MAINNET_USDC;
+            toTokens[4] = POLYGON_MAINNET_USDC;
+
+            bridgeRoutes[0] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: LINEA_MAINNET_BRIDGED_USDC,
+                pctFee: 420000000000000, // 0.042%
+                flatFee: 120000 // 0.12 USDC
+            });
+            bridgeRoutes[1] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: LINEA_MAINNET_BRIDGED_USDC,
+                pctFee: 190000000000000, // 0.019%
+                flatFee: 80000 // 0.08 USDC
+            });
+            bridgeRoutes[2] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: LINEA_MAINNET_BRIDGED_USDC,
+                pctFee: 48000000000000000, // 4.8%
+                flatFee: 9500000 // 9.5 USDC
+            });
+            bridgeRoutes[3] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: LINEA_MAINNET_BRIDGED_USDC,
+                pctFee: 770000000000000, // 0.077%
+                flatFee: 120000 // 0.12 USDC
+            });
+            bridgeRoutes[4] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: LINEA_MAINNET_BRIDGED_USDC,
+                pctFee: 240000000000000, // 0.024%
+                flatFee: 90000 // 0.09 USDC
+            });
+        } else if (block.chainid == LISK_MAINNET) {
+            // TODO
+        } else if (block.chainid == MODE_MAINNET) {
+            // TODO
+        } else if (block.chainid == OP_MAINNET) {
+            chainIds = new uint256[](1);
+            toTokens = new address[](1);
+            bridgeRoutes = new DaimoPayAcrossBridger.AcrossBridgeRoute[](1);
+
+            chainIds[0] = LINEA_MAINNET;
+
+            toTokens[0] = LINEA_MAINNET_BRIDGED_USDC;
+
+            bridgeRoutes[0] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: OP_MAINNET_USDC,
+                pctFee: 720000000000000, // 0.072%
+                flatFee: 180000 // 0.18 USDC
+            });
+        } else if (block.chainid == POLYGON_MAINNET) {
+            chainIds = new uint256[](1);
+            toTokens = new address[](1);
+            bridgeRoutes = new DaimoPayAcrossBridger.AcrossBridgeRoute[](1);
+
+            chainIds[0] = LINEA_MAINNET;
+
+            toTokens[0] = LINEA_MAINNET_BRIDGED_USDC;
+
+            bridgeRoutes[0] = DaimoPayAcrossBridger.AcrossBridgeRoute({
+                localToken: POLYGON_MAINNET_USDC,
+                pctFee: 720000000000000, // 0.072%
+                flatFee: 180000 // 0.18 USDC
+            });
+        } else if (block.chainid == REDSTONE_MAINNET) {
+            // TODO
```

Comment:
replace todos with `revert("Unsupported chain")` -> will catch if we ever try to deploy to those chains before filling in data